fdbserver/workloads/ReadWrite.actor.cpp

```diff
@@ -324,7 +324,14 @@ void ReadWriteCommon::getMetrics(std::vector<PerfMetric>& m) {
 }
 
 Value ReadWriteCommon::randomValue() {
-	return StringRef((uint8_t*)valueString.c_str(), deterministicRandom()->randomInt(minValueBytes, maxValueBytes + 1));
+	int length = deterministicRandom()->randomInt(minValueBytes, maxValueBytes + 1);
+	int zeroPadding = static_cast<int>(0.15 * length);
```

Comment:
Can you make this 0.15 as a knob? In case in future if we want to try with a different number, it will be easy.